src/benchmarks/real-world/ILLink/Utilities.cs

```diff
@@ -61,7 +61,7 @@ public static string PublishSampleProject(string projectFilePath, params string[
         {
             string outputDirectory = GenerateTempFolder();
             Directory.CreateDirectory(outputDirectory);
-            ProcessStartInfo processStartInfo = new ProcessStartInfo("dotnet", $"publish {projectFilePath} -r {CurrentRID} --self-contained -o {outputDirectory} {extraArgs.Aggregate("", (agg, val) => agg + " " + val)}");
+            ProcessStartInfo processStartInfo = new ProcessStartInfo("dotnet", $"publish {projectFilePath} -f {Environment.GetEnvironmentVariable("PERFLAB_TARGET_FRAMEWORKS") ?? "net9.0"} -r {CurrentRID} --self-contained -o {outputDirectory} {extraArgs.Aggregate("", (agg, val) => agg + " " + val)}");
```

Comment:
Will we forget about updating the `net9.0` here next year to keep it in sync? :)

Reply:
Added a comment to the csproj change location to point out this update location. If you have other preferences please let me know!

Reply:
Excellent idea!